Semantics.md

```diff
@@ -678,6 +829,51 @@ outside the range which rounds to an integer in range) traps.
     This trap is intended to be impossible for user code to catch or handle, even in the future when it may be possible to
     handle some other kinds of traps or exceptions.
 
+## Thread operators
+
+  * `is_lock_free`: Given an operand `N`, if the atomic step of an atomic
+    primitive (see [Atomic Memory Accesses](#atomic-memory-accesses)) on a
+    datum of size `N` bytes will be performed without the agent acquiring a
+    lock outside the `N` bytes comprising the datum, then return `1`.
+    Otherwise, return `0`.
```

Comment:
AIUI, we use `operand` to mean stack value, correct? I could be more explicit if desired, though. Tried to clear up the other stuff, PTAL.